git-find-lfs-extensions.py

```diff
@@ -111,3 +123,10 @@ def print_line(ext, share_large, count_large, count_all, size_all, min, max):
             int(result[ext]['min']),
             int(result[ext]['max'])
         )
+
+print("\nAdd to .gitattributes:\n")
+for ext in sorted(result, key=lambda x: (result[x]['type'], x)):
+    if len(ext) > 0 and result[ext]['type'] == "binary" and result[ext]['count_large'] > 0:
+        print('*.{} filter=lfs diff=lfs merge=lfs -text'.format(
+            "".join(c if ('0' <= c <= '9') else "[" + c.upper() + c.lower() + "]" for c in ext)
+        ))
```

Comment:
It think it would great to continue this discussion on the list. I think you don't need to be subscribed in order to reply. Just scroll down and follow the reply instructions:
http://public-inbox.org/git/xmqqzim2vp9e.fsf@gitster.mtv.corp.google.com/